scripts/oddities/OddityProvider.js

```diff
@@ -21,4 +21,6 @@ export const getOddities = () => {
     )
 }
 
-
+export const getOddityById = oddityId => {
```

Comment:
I think we should call these useOddityById, useParkById ... since get implies that a fetch call is invoked.